examples/tutorial-standalone/variables.tf

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "org_id" {
+  description = "The numeric organization id."
+}
+
+variable "folder_id" {
+  description = "The folder to deploy in."
+}
+
+variable "billing_account" {
+  description = "The billing account id associated with the projects, e.g. XXXXXX-YYYYYY-ZZZZZZ."
+}
+
+variable "access_context_manager_policy_id" {
+  description = "The id of the default Access Context Manager policy. Can be obtained by running `gcloud access-context-manager policies list --organization YOUR-ORGANIZATION_ID --format=\"value(name)\"`."
+  type        = string
+  default     = ""
+}
+
+variable "terraform_service_account" {
+  description = "The email address of the service account that will run the Terraform code."
+  type        = string
+}
+
+variable "perimeter_additional_members" {
+  description = "The list of all members to be added on perimeter access, except the service accounts created by this module. Prefix user: (user:email@email.com) or serviceAccount: (serviceAccount:my-service-account@email.com) is required."
+  type        = list(string)
```

Comment:
So, in the example, I'm guessing we force the user to add their own ID?  Either way makes sense, but wanted to confirm that  I thought it was optional in main and we're addressing with the doc/faqs (or we could use that to address - if you don't see anything you recently deployed);  If so, either the tf.vars or main.tf should indicate what the user should specify.